packages/ledger-qldb/src/MockQLDBLedger.ts

```diff
@@ -9,9 +9,14 @@ import {
 
 import { AnchoredDataSerializer } from '@sidetree/core';
 import moment from 'moment';
-import sqlite3 from 'sqlite3';
 import crypto from 'crypto';
 
+let sqlite3: any;
+if (process.env.IMPORT_SQLITE === 'true') {
```

Comment:
Hopefully we can find a better work around for this.

Reply:
Yeah I spent a good few hours trying to find a better way of doing this, tweaking nextjs webpack config, and modifying the package dependencies, conditionally for server, to no avail.  It has to do with this error when indirectly importing sqlite3 on nextjs.  This is the error I get if I don't do this and I try to create or resolve a photon did.  ```(node:86067) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
error - ../ledger-qldb/node_modules/sqlite3/node_modules/node-pre-gyp/lib/pre-binding.js (18:0) @ Object.exports.find
Error: package.json does not exist at /package.json
null```